workflow/rocoto_funcs/base.py

```diff
@@ -50,6 +50,7 @@ def header_entities(xmlFile,expdir):
   mesh_name=os.getenv('MESH_NAME','na3km')
   keepdata=os.getenv('KEEPDATA','yes')
   mpi_run_cmd=os.getenv('MPI_RUN_CMD','srun')
+  wgf=os.getenv('WGF','det')
```

Comment:
what does `wgf` stand for?

Reply:
Thanks, @BenjaminBlake-NOAA !

@hu5970 I would suggest we add a comment on line 53 or other appropriate locations
`wgf=os.getenv('WGF','det')  # WGF: Working Group Function, det, enkf, enfs, firewx`

Reply:
I forgot to commit changes for "exp/exp.jet_conus3km". I added this explanation for WGF in that configure file.